src/startupscriptgenerator/src/dotnetcore/main.go

```diff
@@ -130,6 +130,9 @@ func main() {
 		}
 
 		common.WriteScript(fullOutputPath, command)
+
+		userRunCommand := common.ParseUserRunCommand(fullAppPath + "/appsvc.yaml")
```

Comment:
We should probably have the `appsvc.yaml` file string as a constant in one of the commons file since it's used across multiple `.go` files and allows us to easily update it across the board if the name of the file ever changes.